src/plugins/http_fetch_operator.py

```diff
@@ -68,15 +74,27 @@ def __init__(
         self.tmp_file = tmp_file  # or make temp file + store path in xcom
         self.output_type = output_type  # default is raw, else specify text i.e.
         self.xcom_tmp_dir_task_ids = xcom_tmp_dir_task_ids
-        self.xcom_tmp_dir_key = xcom_tmp_dir_key
+        self.xcom_key = xcom_key
+        self.xcom_token_task_ids = xcom_token_task_ids
         self.verify = verify
 
         super().__init__(*args, **kwargs)
 
     def execute(self, context: Dict) -> None:  # noqa: C901
```

Comment:
`Context` heeft zelfs netjes een eigen type in Airflow:
    from airflow.models.taskinstance import Context